### ISSUE TITLE
LateNight:  preview deck needs label

### ISSUE THREAD

**Issue by Holzhaus** [Holzhaus/mixxx-gh-issue-migration#4178]
I'm filing this as a question since it's not really a bug.
If approved, I will file a bug report for this.

I find the preview deck section a little lost in all skins.
Wouldn't it be prettier to have a title on it.
And also more practical because when empty, it's not clear what that section is about.

Of course, we need all available space too, so I would suggest to show the title "Preview Deck" in the track-title space when no track is loaded.
When a track is loaded, the "Preview Deck" title can disappear, from that moment it's quite clear what the section is about.
Basically, it's identical to how the search box operates.
